ocaml/idl/datamodel.ml

```diff
@@ -9200,6 +9202,50 @@ let alert =
     ()
 *)
 
+let net_sriov_create = call
+    ~name:"create"
+    ~doc:"Create a net-sriov on specified pif"
```

Comment:
`Create a net-sriov on a specified PIF`. Note the capitalization of VIF, PIF and so on in documentation.

Reply:
yes